packages/InputGroup/style.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components'
+
+export const LabelStyle = styled.label(() => ({
+  span: {
+    fontSize: '14px',
+    color: '#2A2C2E',
+    display: 'block',
+  },
+}))
+
+export const InputGroupStyle = styled.div(({ hasValue }) => ({
```

Comment:
There is meant to be a max-width of 768px on this component, and a min-width of 384px, as per ACs in #228

Reply:
Changed